### ISSUE TITLE
Internal anal sphincter achalasia

### ISSUE THREAD

**Issue by jagcgc** [obophenotype/human-phenotype-ontology#10235]
**Preferred term label:**
Internal anal sphincter achalasia

**Synonyms**
- Anal achalasia

**Definition (free text, please give PubMed ID)**
PMID: 19782306
Internal anal sphincter achalasia is a clinical condition with presentation similar to Hirschsprung's disease, but with the presence of ganglion cells on rectal suction biopsy. It is caused by failure of relaxation of the internal sphincter,

PMID: 33301220
Currently, the diagnosis of internal anal sphincter achalasia is made when the rectoanal inhibitory reflex is absent during anorectal manometry but ganglion cells are present on rectal biopsy. Patients with internal anal sphincter achalasia appear to represent a more severe patient population compared to patients with functional constipation (earlier onset of symptoms, more likely to have redundancy or segmental dilation of the colon), but with later symptom onset compared to patients with Hirschsprung disease.

**Parent term (use hpo.jax.org/app)**
Abnormal anus physiology HP:0034916

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
None to my knowledge

**Your nano-attribution (ORCID)**
0000-0001-8481-3559

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000018
- label: Internal anal sphincter achalasia
- parents: HP:0034916
- pmid: PMID:19782306|PMID:33301220
- def: A disorder of the internal anal sphincter (IAS) in which the IAS fails to relax that manifests clinically as intractable constipation.
- comment: The internal anal sphincter, which is a specialized smooth muscle continuation of the circular muscle layer of rectum, plays a significant role in the maintenance of anorectal continence and in the pathophysiology of incontinence and constipation. Internal anal sphincter achalasia is a clinical condition with presentation similar to Hirschsprung disease but with the presence of ganglion cells on suction rectal biopsy.
- orcid: orcid:0000-0001-8481-3559
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10235
- synonyms: Anal achalasia(NONE)
